examples/petersen.jl

```diff
@@ -1,22 +1,62 @@
+# for mapping to a grid graph
 using UnitDiskMapping, Graphs
+# for solving the MIS
+using GenericTensorNetworks
 
-function petersen_graph()
-    graph = SimpleGraph(10)
-    for (i, j) in [(1,2), (1,4), (1,9), (2,3), (2, 7), (3,5), (3, 10), (4, 5), (4,6), (5, 8), (6, 7), (6,10), (7,8),(8,9),(9,10)]
-        add_edge!(graph, i, j)
+########## The 5-vertex graph ##########
+# the five vertex demo graph in the paper: "COMPUTER-ASSISTED GADGET DESIGN AND PROBLEM REDUCTION OF UNWEIGHTED MAXIMUM INDEPENDENT SET"
+function demograph()
+    g = SimpleGraph(5)
+    for (i, j) in [(1, 2), (2, 4), (3, 4), (1, 3), (4, 5), (1, 5)]
+        add_edge!(g, i, j)
     end
-    @assert all(==(3), degree.(Ref(graph), 1:10))
-    return graph
+    return g
 end
 
-g = petersen_graph()
-ug = embed_graph(g)
-G, tape = apply_gadgets!(copy(ug))
-locs = coordinates(G)
+# create the source graph
+g = demograph()
 
-using GenericTensorNetworks
-s1 = solve(IndependentSet(g), SizeMax())
-s2 = solve(IndependentSet(SimpleGraph(G)), SizeMax())
-mis_overhead0 = 2 * nv(g) * (nv(g)-1) + nv(g)
-mis_overhead1 = sum(x->mis_overhead(x[1]), tape)
-s1[].n == s2[].n - mis_overhead0 - mis_overhead1
\ No newline at end of file
+# map it to a grid graph (unweighted)
+# the vertex order is optimized with the Branching path decomposition algorithm
+mapres = UnitDiskMapping.map_graph(g; vertex_order=Branching())
+
+# visualize the mapped graph in terminal or in a plotting plane
+print(mapres.grid_graph)
+show_graph(mapres.grid_graph)
+
+# solve the MIS size of the source graph
+s1 = solve(IndependentSet(g), SizeMax())[]
+# solve the MIS size of the mapped graph
+s2 = solve(IndependentSet(SimpleGraph(mapres.grid_graph)), SizeMax())[]
+# show that the overhead is correct
+s1.n == s2.n - mapres.mis_overhead
+
+# the solution of the mapped graph
+mapped_mis = solve(IndependentSet(SimpleGraph(mapres.grid_graph)), SingleConfigMax())[].c.data
+# plot the solution
+show_config(mapres.grid_graph, mapped_mis)
+# map the solution back
+original_mis = UnitDiskMapping.map_config_back(mapres, collect(mapped_mis))
+# show that it is an MIS
```

Comment:
Does this show it is a MIS or just an independent set? Also, perhaps change to "a MIS" instead of "an MIS" to be consistent.